lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -2803,9 +2801,6 @@ void mme_app_handle_nw_init_ded_bearer_actv_req(
       "ue_id " MME_UE_S1AP_ID_FMT "\n",
       ue_context_p->mme_ue_s1ap_id);
   }
-  free_wrapper((void **) &activate_ded_bearer_req.tft);
-  free_wrapper((void **) &activate_ded_bearer_req.pco);
```

Comment:
where do we free these later?

Reply:
Memory allocated to tft in mme_app_handle_nw_init_ded_bearer_actv_req() function is assigned to the tft in esm_ebr_context when a new context gets created for the bearer in esm_ebr_context_create(). This gets freed in esm_ebr_context_release() ->free_traffic_flow_template(). The above free_wrapper function was redundant. Even though it was called it was not freeing the tft because before invoking this function the pointer was set to NULL in _emm_cn_activate_dedicated_bearer_req() function. Also, this is the wrong place to free as the same memory is assigned to the tft in esm_ebr_context.